engine/src/cython/io.cpp

```diff
@@ -71,9 +71,8 @@ TableSchema parseSchema(std::vector<std::string> files,
 		// TODO percy cudf0.12 port to cudf::column
 		//GDFRefCounter::getInstance()->deregister_column(column_cpp.get_gdf_column());
 		//tableSchema.columns.push_back(column_cpp.get_gdf_column());
-		
-		tableSchema.names.push_back(column_cpp.name());
 	}
+	tableSchema.blazingTableView.setNames(schema.get_names());
```

Comment:
views are immutable. I see why youa re doing this. You need to make a view and if theres no table behind it that doesnt really work. And we cant really create a table since there is no where to put it to keep it in scope.